contrib/examples/actions/workflows/orquesta-ask-multiple-parameters.yaml

```diff
@@ -0,0 +1,56 @@
+version: 1.0
+
+description: A basic workflow that demonstrates inquiry with multiple parameters.
+
+tasks:
+  start:
+    action: core.echo message="Automation started."
+    next:
+      - when: <% succeeded() %>
+        do: get_approval
+
+  get_approval:
+    action: core.ask
+    input:
+      schema:
+        type: object
+        properties:
+          approved:
+            type: boolean
+            description: "Continue?"
+            required: True
+          department_id:
+            type: number
+            description: "Your department ID:"
+            required: True
+          extra_output:
+            type: string
+            description: "Your message to echo next if you approve to continue:"
+            default: "None"
+      ttl: 60
+    next:
+      - when: <% task(get_approval).result.response.extra_output != 'None' and task(get_approval).result.response.approved = true %>
```

Comment:
If you use NoneType, I believe the check would be `task(get_approval).result.response.extra_output != null`

Reply:
I will test this latest tomorrow and come back with a working solution.

Reply:
This test was not successful. Only number, string and booleans seem to be supported. Used .containsKey() for this check.

Reply:
This test failed. Only boolean, string and names are supported types. I used .containsKey() for this check in the updated code.